### DISCUSSION TITLE
Opacity feature in timeline layers...

### DISCUSSION THREAD

**Discussion by cvanalphen** [keiwando/toonsquid#490] [General]
Hi,

Maybe this option is already there but I can't find it (also in the manual)... 

Is it currently possible to adjust opacity for a layer in the timeline? This will affect all drawings that belongs to that layer. I currently have to go to each  individual drawing property to adjust the opacity which is very time consuming... 

Thanks!
Toff

**Comment by cvanalphen** [comment]
Oh wait... I think I understand now... Maybe it was designed this way because users can convert a layer to a symbol and from there they can adjust the opacity...

**Comment by keiwando** [comment]
In the layer inspector, you can switch between properties of the selected drawing layer and the selected animation layer:
https://toonsquid.com/handbook/layers/inspector/

The animation layer has an opacity property that does what you are looking for while keeping all of your drawings in the main timeline. No need to create a symbol.

**Reply by cvanalphen** [reply]
Fantastic!!! Thanks a lot for answering so fast🙏 Really helpful 😊
